packages/suite/src/views/settings/index.tsx

```diff
@@ -209,22 +208,39 @@ const Settings = () => {
                         description={<Translation id="TR_LABELING_FEATURE_ALLOWS" />}
                     />
                     <ActionColumn>
-                        <Switch
-                            // hmm maybe it should never be disabled, as it is not device related option (although it triggers device flow?)
-                            // but on the other hand there still may be case when it remembered device is disconnected and its metadata.status
-                            // is cancelled or disabled. In such case, initMetadata does not make any sense as it needs device connected.
-                            // You could say: "ok, whatever, but this switch is changing only application setting, why messing with device?"
-                            // Yes, you are right, but if it was done this way, you would enable metadata, then go to wallet, discovery
-                            // and maybe device authorization would be triggered and only after that you would get metadata flow started, wouldn't
-                            // that be confusing? I believe it is better to do it right away, but need for disabling this switch in specific
-                            // edge case is a drawback.
-                            isDisabled={!metadata.enabled && !device?.connected && isDeviceLocked}
-                            data-test="@settings/metadata-switch"
-                            checked={metadata.enabled}
-                            onChange={() =>
-                                metadata.enabled ? disableMetadata() : initMetadata(true)
+                        <Tooltip
+                            maxWidth={280}
+                            offset={10}
+                            placement="top"
+                            dashed={false}
+                            cursor="default"
```

Comment:
cursor should be `not-allowed` according the https://github.com/trezor/trezor-suite/issues/4728#issue-1101387663

Reply:
still not working 🤔 react-switch has defined cursor: default

Reply:
@pingpwned still not fixed, right?

Reply:
it is possible to achieve custom cursor when disabled with absolute positioned pseudo element with higher z-index.

Reply:
it is possible, but not resolved, right?
https://user-images.githubusercontent.com/3729633/152010201-a8718a63-0a77-4c51-a44e-dac7b53ac7f5.mov

Reply:
added in 66999f3. Also added cursor: not-allowed for disabled ActionButtons